Makefile

```diff
@@ -3,6 +3,18 @@ default: build
 build:
 	docker build -t image-builder-rpi .
 
+get-cluster-lab-images:
+	mkdir -p builder/files/var/local/
```

Comment:
What about adding a shell script doing all these steps? Then we have a place to put the version numbers into. So someone can build it without Travis locally.